Formula/kubernetes-service-catalog-client.rb

```diff
@@ -0,0 +1,28 @@
+class KubernetesServiceCatalogClient < Formula
+  desc "Consume Services in k8s using the OSB API"
+  homepage "https://svc-cat.io/"
+  url "https://github.com/kubernetes-incubator/service-catalog.git",
+      :tag => "v0.1.23",
+      :revision => "87a5db0e1e0359ce372037a235ce4448944c8611"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["NO_DOCKER"] = "1"
+    dir = buildpath/"src/github.com/kubernetes-incubator/service-catalog"
+
+    dir.install buildpath.children
+
+    cd dir do
+      ldflags = "-s -w -X github.com/kubernetes-incubator/service-catalog/pkg.VERSION=v#{version}"
```

Comment:
This should be wrapped to 80 characters, ideally.

Reply:
Thanks @DomT4 . I tried to bring it below 80 characters but the url is too long for that. The only way would be for me to break the url and would not be good. Let me know if it is okay to just remain as such.

Reply:
Thanks @DomT4. Changed it as requested. Nice idea.